RHEL6_7/services/openldap/check

```diff
@@ -55,22 +55,22 @@ If you are using the bdb or hdb openldap back ends, it is necessary to update
 the database.
 
 Back up the data:
-`# slapcat -l backup.ldif`
+# slapcat -l backup.ldif
 
-For more information about slapcat, see the slapcat(8) man pages.
+For more information about slapcat, see the slapcat(8C) man pages.
```

Comment:
The slapcat manpage indeed has `8C` in header. However, it looks like that part is just visual; the real section is still `8`, not `8C`:

    # man 8c slapcat
    No entry for slapcat in section 8c of the manual
    # find /usr/share/man | grep slapcat
    /usr/share/man/man8/slapcat.8.gz
    # man 8 slapcat
    ...(man page is shown)...

(I wonder why the `C` is there in the first place; it looks like a bug to me.)

Also, note that alt. manpage sections are spelled with lowercase letters:

    # man 1P unlink
    No entry for unlink in section 1P of the manual
    # man 1p unlink
    ...(man page is shown)...

They do appear as uppercase in the header but that's because the whole header field is actually uppercase: `UNLINK(1P)`.

Reply:
(note that there are more instances of this)

Reply:
Poor cat, everyone's harsh on her.